include/picongpu/plugins/radiation/Radiation.hpp

```diff
@@ -1250,6 +1238,53 @@ class Radiation : public ISimulationPlugin
 
 } // namespace radiation
 } // namespace plugins
+
+namespace particles
+{
+namespace traits
+{
+    template<
+            typename T_Species,
+            typename T_UnspecifiedSpecies
+    >
+    struct SpeciesEligibleForSolver<
+            T_Species,
+            plugins::radiation::Radiation< T_UnspecifiedSpecies >
+    >
+    {
+    using FrameType = typename T_Species::FrameType;
+
+    // this plugin needs the position, weighting, momentum and momentumPrev1 to run
+    using RequiredIdentifiers = MakeSeq_t<
+            position<>,
```

Comment:
It looks like the template arguments are on the right level of indentation, but `using RequiredIdentifiers` is not, same for other types below.